dbt_subprojects/solana/models/_sector/gas/gas_backfills/tx_fees/gas_solana_tx_fees_current.sql

```diff
@@ -1,14 +1,14 @@
 {{ config(
     schema = 'gas_solana',
     alias = 'tx_fees_current',
-    partition_by = ['block_date', 'block_hour'],
+    partition_by = ['block_date'],
     materialized = 'incremental',
     file_format = 'delta',
     incremental_strategy = 'delete+insert',
```

Comment:
do want to get rid of the non atomic delete+insert while we're doing the refresh anyway?

Reply:
good point, updated